unlock-app/src/components/lock/Lock.js

```diff
@@ -47,10 +48,18 @@ export const Lock = ({
         <BalanceProvider
           amount={lock.keyPrice}
           render={(ethPrice, fiatPrice) => (
-            <LockBody>
-              <EthPrice>{ethPrice} Eth</EthPrice>
-              <FiatPrice>${fiatPrice}</FiatPrice>
-            </LockBody>
+            <div>
```

Comment:
Did you have a rebase issue? Is that part of your PR?

Reply:
Oh that's extraordinarily frustrating. No, it isn't. Stand by.

Reply:
OK. This is fixed now.